doc/release-notes/10494-payara-upgrade.md

```diff
@@ -0,0 +1,112 @@
+# Upgrade Payara to v6.2024.6
+
+With this version of Dataverse, we encourage you to upgrade to version 6.2024.6.
+This will address security issues accumulated since the release of 6.2023.8, which was required since Dataverse release 6.0.
+
+## Instructions for Upgrading
+
+If you are using GDCC containers, this upgrade is included when pulling new release images.
+No manual intervention is necessary.
+
+We recommend you ensure you followed all update instructions from the past releases regarding Payara.
+(Latest Payara update was for [v6.0](https://github.com/IQSS/dataverse/releases/tag/v6.0))
+
+Upgrading requires a maintenance window and downtime. Please plan ahead, create backups of your database, etc.
+
+The steps below are a simple matter of reusing your existing domain directory with the new distribution.
+But we also recommend that you review the Payara upgrade instructions as it could be helpful during any troubleshooting:
+[Payara Release Notes](https://docs.payara.fish/community/docs/Release%20Notes/Release%20Notes%206.2024.6.html)
+We assume you are already on a Dataverse 6.x installation, using a Payara 6.x release.
+
+```shell
+export PAYARA=/usr/local/payara5
+```
+
+(or `setenv PAYARA /usr/local/payara6` if you are using a `csh`-like shell)
+
+1\. Undeploy the previous version
+
+```shell
+    $PAYARA/bin/asadmin list-applications
+    $PAYARA/bin/asadmin undeploy dataverse<-version>
+```
+
+2\. Stop Payara
+
+```shell
+    service payara stop
+    rm -rf $PAYARA/glassfish/domains/domain1/generated
+    rm -rf $PAYARA/glassfish/domains/domain1/osgi-cache
+    rm -rf $PAYARA/glassfish/domains/domain1/lib/databases
+```
+
+3\. Move the current Payara directory out of the way
+
+```shell
+    mv $PAYARA $PAYARA.MOVED
+```
+
+4\. Download the new Payara version (6.2024.6), and unzip it in its place
+
+5\. Replace the brand new payara/glassfish/domains/domain1 with your old, preserved domain1
+
+6\. Make sure that you have the following `--add-opens` options in your domain.xml. If not present, add them:
```

Comment:
I think these all have to be there by now, so this step isn't needed?

Reply:
In a 6.2023.8 domain file these are not included. So IMHO folks upgrading should make sure they have them when copying their old domain over.

Reply:
Ah - I see. There are only 4 new entries (repeated twice in the two configs) and the rest is just from the diff showing things before/after the changes.

Reply:
That's why I used the `diff` highlighter, so it  becomes more obvious when you look at it in the final notes doc/release page.